components/user/UserProfile.js

```diff
@@ -76,15 +76,19 @@ function UserProfile({ BASE_URL, data }) {
       {!qrShow && (
         <div className="flex flex-wrap justify-center">
           {data.tags?.length > 0 &&
-            data.tags.map((tag) => (
-              <Tag
-                name={tag}
-                key={tag.toLowerCase()}
-                onClick={() =>
-                  router.push(`/search?keyword=${tag.toLowerCase()}`)
-                }
-              />
-            ))}
+            data.tags.map((tag, index) => {
+              const trimmed_tag = tag.trim();
```

Comment:
the naming does not follow the project standards, it should be `trimmedTag`

Reply:
I am sorry. I will change this.